wallet_data/src/commonMain/kotlin/net/thechance/mena/wallet/data/mapper/PaymentMapper.kt

```diff
@@ -0,0 +1,12 @@
+package net.thechance.mena.wallet.data.mapper
+
+import net.thechance.mena.wallet.data.dto.PaymentConfirmationDto
+import net.thechance.mena.wallet.domain.model.PaymentConfirmation
+import net.thechance.mena.wallet.domain.model.TransactionStatus
+
+fun PaymentConfirmationDto.toEntity() = PaymentConfirmation(
+    balance = currentBalance?: 0.0,
+    receiverName = recipientName ?: "",
+    receiverImg = recipientImageUrl,
+    status = if (isValid == true) TransactionStatus.SUCCESS else TransactionStatus.FAILED
```

Comment:
Why not just use isValid as boolean in the domain model? I don't think transaction status corresponds one to one to payment confirmation state. For example, if we added pending status for transactions can a payment confirmation be pending? A boolean value is sufficient here i think.
```suggestion
    isValid = isValid == true
```